bot/event_handler.py

```diff
@@ -83,7 +90,7 @@ def _handle_message(self, event):
 
         intent_value = intent_entity['value']
         if intent_value in intents:
-            intents[intent_value](self.msg_writer, event, wit_resp['entities'])
+            intents[intent_value][0](self.msg_writer, event, wit_resp['entities'])
```

Comment:
It's generally not best practice to reference indices directly since they could change. At minimum you could change from a tuple to a dict and reference by the key, though there may be a more pythonic approach.